### DISCUSSION TITLE
Margin notes only in book chapter

### DISCUSSION THREAD

**Discussion by dominicwhite** [quarto-dev/quarto-cli#5122] [Q&A] [answered]
I am writing a Quarto book split over multiple chapter files.

I am trying to add margin notes following this guide here: [https://quarto.org/docs/blog/posts/2022-02-17-advanced-layout/](https://quarto.org/docs/blog/posts/2022-02-17-advanced-layout/)

If I add
```
reference-location: margin
citation-location: margin
```
to the pdf format section my `_quarto.yml` file for the whole book then I get footnotes correctly rendered in the margins in the individual chapters.

_However_, it also forces other sections, like the title page and the table of contents, to have that same margin, which I do not want.

I would like to have the margin space added just in the chapters (and not on the title page or table of contents), but when I add the two settings above to the top of individual chapter files instead, either using the recommended style from the blog post:

```
---
reference-location: margin
citation-location: margin
---
```

or as something like this:

```
---
format:
  pdf:
    documentclass: scrreport
    reference-location: margin
    citation-location: margin
---
```

then I get regular footnotes instead of margin notes.

Is it possible to turn on margin notes only for certain chapters, and if so, what I am doing wrong with my settings?

**Comment by dragonstyle** [comment, marked as answer]
Right, geometry is defined globally (include for the TOC and other 'meta' type pages). You could experiment with trying to override this directly with included LaTeX that sets page geometry globally:

```markdown
geometry:
  - top=30mm
  - left=20mm
```

which sets geometry for the `geometry` package <https://ctan.math.utah.edu/ctan/tex-archive/macros/latex/contrib/geometry/geometry.pdf>. When you define this, we should stop trying to adjust the dependency and allow you to control things.

There are ways to set per page geometry using Latex, so it might be possible to define a global geometry that affects all the pages, then override that for the chapters, though you'd definitely be doing something a bit more advanced with LaTeX and something I don't really have experience with...

I'm going to convert this to an issue as well to consider whether we should just automatically set default page geometry for the title page and TOC (it seems to be we should be doing this).

**Reply by dominicwhite** [reply]
Thanks for the detailed reply. I played around with the geometry package as you suggested, and was able to override the geometry with your suggested settings, and then set a wider right margin geometry starting in the first chapter by adding the `\newgeometry` command at the top of the first chapter's `.qmd` file.

While this does allow me to override the geometry, it unfortunately doesn't play well with margin notes.

For example, here is the default appearance of the margin notes (they occupy a fairly wide fraction of the page that makes them easy to read):
![Screenshot from 2023-04-14 10-14-07](https://user-images.githubusercontent.com/6020835/232076396-df6d1b06-a592-477f-b10d-4c46ae62d3ff.png)

versus here is how they appear when I override the global geometry with the settings you suggested and then later use `\newgeometry` to set a 3 inch right margin for the chapter (at which point the margin notes become very compressed - possibly something about how the ratio of content to margin note is determined?):
![Screenshot from 2023-04-14 10-41-36](https://user-images.githubusercontent.com/6020835/232076918-25ae2d92-93b7-4196-b4a0-fbf8b3e0d0d6.png)

I shall probably go back to regular footnotes for the time being (to avoid the trap of spending more time on Latex settings than actually writing), but I shall watch the progress of Issue #5143 with interest. Thanks for your work on this project!

**Reply by dragonstyle** [reply]
Thanks for sharing your experience - in general the margin notes should respect the geometry of their container - I'm not sure why that is happening and appreciate that you may not want to spend more time on this! If you do want to tinker further, you can see the layout lines created by the geometry by using:

```yaml
format:
  pdf:
    geometry:
      - showframe     
```

I don't recall exactly how we manage geometry for books specifically, but there is some special handling for 'sidedness' of the even vs odd pages of books that may be throwing a wrench into things...
